dbt/include/mssql/macros/indexes.sql

```diff
@@ -0,0 +1,76 @@
+{% macro drop_all_indexes_on_table() %}
```

Comment:
This is a great start, but is a giant function. While it may make sense to have a "drop all" function, this is large and is actually doing multiple things (dropping different kinds of indexes). I'd like to see each of these steps moved into their own functions, all of which can be called from the "drop all" function.

Reply:
Great feedback!  Will do.  May be a few days before I get to it.